cli-output/src/cli_output.rs

```diff
@@ -231,12 +231,8 @@ pub struct CliSlotStatus {
 pub struct CliEpochInfo {
     #[serde(flatten)]
     pub epoch_info: EpochInfo,
-}
-
-impl From<EpochInfo> for CliEpochInfo {
-    fn from(epoch_info: EpochInfo) -> Self {
-        Self { epoch_info }
-    }
+    #[serde(skip)]
+    pub avg_slot_time_ms: u64,
```

Comment:
My eye would prefer to see `avg` spelled out.  
```suggestion
    pub average_slot_time_ms: u64,
```